src/__tests__/api_success.ts

```diff
@@ -46,20 +49,36 @@ test.before((t) => {
   t.context = {
     server: https
       .createServer(opts, (req, res) => {
-        switch (req.url) {
-          case evaluationAPI:
-            res.setHeader('Content-Type', 'application/json');
-            res.end(JSON.stringify(dummyEvalResponse));
-            break;
-          case eventsAPI:
-            res.setHeader('Content-Type', 'application/json');
-            res.end(JSON.stringify(dummpyRegisterEvtsResponse));
-            break;
-          default:
-            res.writeHead(400);
-            res.end();
-            break;
-        }
+        let body = '';
+
+        req.on('data', (chunk) => {
+          body += chunk.toString();
+        });
+
+        req.on('end', () => {
+          try {
+            let jsonBody = JSON.parse(body) as BaseRequest;
+            // Verify the request needs to inclued `sdkVersion` and `sourceId`
```

Comment:
```suggestion
            // Verify the request needs to include `sdkVersion` and `sourceId`
```

Reply:
I updated in https://github.com/bucketeer-io/node-server-sdk/pull/58/commits/44776a767cb72de834e5fffb37a7e41ba31ad743